### DISCUSSION TITLE
How to create suneditor.min.js?

### DISCUSSION THREAD

**Discussion by gualtieronovo** [JiHong88/SunEditor#1362] [Q&A]
How can I create  suneditor.min.js from source code?

**Comment by last-resort** [comment]
1. You need to have npm installed
2. Clone the git project in your IDE or with git clone https://github.com/JiHong88/SunEditor.git
3. Navigate to the root folder of the project
4. Run npm install 
5. Run npm run build:prod  if  you are on V3 branch or npm run build for V2
6. You'll se a dist folder in the project with the min.js and min.css inside
